package.json

```diff
@@ -13,7 +13,7 @@
     "web-vitals": "^1.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
Mr @hiderbrandon I wonder if that openssl was truly necessary ?

Reply:
teacher the base code didn't run the "npm run start " on my computer , probabtly it is not necesasary sorry i just was hurried i could'nt check other way

Reply:
Gotcha.

Reply:
Mr @hiderbrandon why some commits came from the @SH-Arias account ?

Reply:
teacher, I just realized about it but I don´t know who SH-arias is, I did the commits from a computer on the EISC ,  probably he has configured his "git remote" on that computer and it threw the "push" from it

the last two "commits" were pushed from my home computer,  i know it could look suspicious but we can contact with SH-arias and corroborate my hypothesis on the EISC's computer

Reply:
No Worries.